From 8d98c70ac6a829b950b756d0c3daf751b44c5390 Mon Sep 17 00:00:00 2001
From: David Hu <david.hu2@arm.com>
Date: Thu, 18 Jul 2024 10:33:43 +0100
Subject: [PATCH] check-sr-results.yaml: Reset SCT failure number to 0

After Update Capsule invalid flag check test failures are fixed, all of
the selected SCT test cases pass on Kronos.
Reset the excepted number of SCT failed test cases to 0 in
check-sr-result.yaml to align with the latest ACS test results.

Upstream-Status: Inappropriate [Kronos-specific update]
Signed-off-by: David Hu <david.hu2@arm.com>
---
 check-sr-results.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/check-sr-results.yaml b/check-sr-results.yaml
index f81a745..6a446ae 100644
--- a/check-sr-results.yaml
+++ b/check-sr-results.yaml
@@ -52,7 +52,7 @@ tree:
         must-contain:
           - SCT Summary
           - Dropped:|0|
-          - Failure:|3|
+          - Failure:|0|
           - Warning:|0|
           - Dropped by group
           - Failure by group
-- 
2.34.1

